sdks/java/extensions/sql/zetasql/src/test/java/org/apache/beam/sdk/extensions/sql/zetasql/ZetaSQLDialectSpecTest.java

```diff
@@ -3587,11 +3585,14 @@ public void testCastToDateWithCase() {
     PCollection<Row> stream = BeamSqlRelUtils.toPCollection(pipeline, beamRelNode);
 
     Schema resultType =
-        Schema.builder().addInt32Field("f_int").addNullableField("f_date", DATETIME).build();
+        Schema.builder()
+            .addInt64Field("f_long")
+            .addNullableField("f_date", FieldType.logicalType(SqlTypes.DATE))
+            .build();
 
     PAssert.that(stream)
         .containsInAnyOrder(
-            Row.withSchema(resultType).addValues(1, parseDate("2018-10-18")).build());
+            Row.withSchema(resultType).addValues(1L, parseDateToLocalDate("2018-10-18")).build());
```

Comment:
I think you don't have to implement a 'parseDateToLocalDate' function yourself. You can use `LocalDate.of(y, m, d)` or 'LocalDate.parse("yyyy-mm-dd")' directly.

Reply:
Thanks!